pkg/solidity-utils/test/FixedPoint.test.ts

```diff
@@ -13,8 +12,8 @@ describe('FixedPoint', () => {
 
   const checkPow = async (x: number, pow: number) => {
     const result = fp(x ** pow);
-    expect(await lib.powDown(fp(x), fp(pow))).to.equal(result);
-    expect(await lib.powUp(fp(x), fp(pow))).to.equal(result);
+    expectEqualWithError(await lib.powDown(fp(x), fp(pow)), result, 0.00000001);
```

Comment:
FixedPoint.pow is supposed to guarantee a max relative error of 10^-14, so we should be able to use that here.